index.html

```diff
@@ -4599,6 +4599,32 @@ <h3>Long-Lived Identifier-Based Correlation</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Metadata-based Correlation</h3>
+
+        <p>
+The use of different extension points described in Section
+<a href="#basic-concepts"></a> and Section <a href="#advanced-concepts"></a>
+can serve as an unintentional or unwanted correlation mechanism if the number
+of <a>issuers</a> using the specific extension type, or combination of extension types,
+is relatively small.
```

Comment:
```suggestion
of <a>issuers</a> using the specific single, or combination of, extension type(s),
is relatively small.
```

Reply:
You can parse the above as "using the combination of extension type is relatively small" (note "extension type", singular) -- the previous language feels easier to parse. Can you provide an alternative that achieves what you want to achieve here?

Reply:
Much better, applied your latest fix.